lib/engines/content_object_store/app/lib/content_object_store/publishable.rb

```diff
@@ -41,6 +41,28 @@ def schedule_with_rollback
       end
     end
 
+    def create_draft_edition(schema)
+      raise ArgumentError, "Local database changes not given" unless block_given?
+
+      ActiveRecord::Base.transaction do
+        content_block_edition = yield
+        content_id = content_block_edition.document.content_id
+        organisation_id = content_block_edition.lead_organisation.content_id
+
+        create_publishing_api_edition(
```

Comment:
do we still need to call `create_publishing_api_edition` up above in `publish`? Because we might be passing in some new updated `details`/`title` when we publish?

Reply:
Oooh, that's a good point actually. I think we should still push a draft up before publishing, but we probably don't need those args there now.